server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -323,6 +323,12 @@ Issues related to authorization may exist just for some other versions of {jdgse
 +
 It is currently required to have different configuration files for the JDG servers on both sites as the Infinispan subsystem does not support replacing site names with expressions. See link:https://issues.redhat.com/browse/WFLY-9458[this issue] for more details.
 
+NOTE: The _PUBLIC_IP_ADDRESS_ below refers to the IP address or hostname, which can be used for your server to bind to. Note that
+every {jdgserver_name} server and {project_name} server needs to use different address. During example setup with all hosts on single laptop,
```

Comment:
s/During example setup with all hosts on single laptop,.../During example setup with all the servers running on the same host,.../g ?

Reply:
Will do. Thanks